From 673150d63e6887503d0c467e13735a24d15ae7aa Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Fri, 20 Jan 2017 07:45:43 +0100
Subject: [PATCH] Move AM_CONDITIONAL(HAVE_DSSIM) outside of the iqa check
 block

or else configure fails with

  error: conditional "HAVE_DSSIM" was never defined.

when iqa is not installed.

Signed-off-by: Lars Wendler <polynomial-c@gentoo.org>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 92411b194..185b49f48 100644
--- a/configure.ac
+++ b/configure.ac
@@ -359,13 +359,13 @@ AG_GST_CHECK_FEATURE(IQA, [iqa], iqa , [
     HAVE_IQA="no"
   ])
 
-  AM_CONDITIONAL(HAVE_DSSIM, test "x$HAVE_DSSIM" = "xyes")
   if test "x$HAVE_DSSIM" = "xyes"; then
     AC_DEFINE(HAVE_DSSIM, 1, [Define if you have dssim library])
   fi
   AC_SUBST(DSSIM_LIBS)
   AC_SUBST(DSSIM_CFLAGS)
 ])
+AM_CONDITIONAL(HAVE_DSSIM, test "x$HAVE_DSSIM" = "xyes")
 
 dnl Orc
 ORC_CHECK([0.4.17])
-- 
2.11.0

